data-raw/DATASET.R

```diff
@@ -54,7 +54,18 @@ df_patient_characteristics <-
   tibble::tibble(
     patient_id = paste("ID", 1:100),
     trt = sample(c("Drug A", "Drug B"), size = 100, replace = TRUE),
-    age = floor(rnorm(100, 50, 10))
+    age = floor(rnorm(100, 50, 10)),
+    marker = abs(-age / 33 + rnorm(100, 7, 1)),
+    status =
+      sample(
+        x = c("Completed Study", "Adverse Event", "Progressive Disease",
+              "Physician Decision", "Subject Withdrew", "Other"),
```

Comment:
```suggestion
              "Physician Decision", "Subject Withdrew", "Active"),
```

The status variable looks mostly like off study reasons - for a more general status variable, how about we change `Other` to `Active`?

Reply:
sounds good to me!